syntax = "proto3";

option go_package = "pb/game";

import "proto/common/tip.proto";
import "proto/common/common.proto";
import "proto/common/empty.proto";

option cc_generic_services = true;

message GameKickPlayerRequest {
  TipInfoMessage reason = 1;
  string operator = 2;
}

service CentreClientPlayerCommonService 
{
  rpc SendTipToClient (TipInfoMessage) returns (Empty);
  rpc KickPlayer (GameKickPlayerRequest) returns (Empty);
}